images/sds-local-volume-csi/src/driver/node.go

```diff
@@ -93,6 +100,68 @@ func (d *Driver) NodeStageVolume(_ context.Context, request *csi.NodeStageVolume
 		return nil, status.Errorf(codes.InvalidArgument, fmt.Sprintf("[NodeStageVolume] Invalid fsType: %s. Supported values: %v", fsType, ValidFSTypes))
 	}
 
+	blockSize, err := recheckFormattingOptionParameter(context, internal.BlockSizeKey, internal.FileSystemConfigs, fsType)
+	if err != nil {
+		return nil, err
+	}
+	inodeSize, err := recheckFormattingOptionParameter(context, internal.InodeSizeKey, internal.FileSystemConfigs, fsType)
+	if err != nil {
+		return nil, err
+	}
+	bytesPerInode, err := recheckFormattingOptionParameter(context, internal.BytesPerInodeKey, internal.FileSystemConfigs, fsType)
+	if err != nil {
+		return nil, err
+	}
+	numInodes, err := recheckFormattingOptionParameter(context, internal.NumberOfInodesKey, internal.FileSystemConfigs, fsType)
+	if err != nil {
+		return nil, err
+	}
+	ext4BigAlloc, err := recheckFormattingOptionParameter(context, internal.Ext4BigAllocKey, internal.FileSystemConfigs, fsType)
+	if err != nil {
+		return nil, err
+	}
+	ext4ClusterSize, err := recheckFormattingOptionParameter(context, internal.Ext4ClusterSizeKey, internal.FileSystemConfigs, fsType)
+	if err != nil {
+		return nil, err
+	}
+
+	formatOptions := []string{}
```

Comment:
Seems like its better to initialize with make([]string,0,7) to avoid mem allocations

Reply:
Agree before update, but after last update it will be empty most of the time, so I'm leaving it as-is.